src/RA_LeaderboardPopup.h

```diff
@@ -18,6 +18,7 @@ class LeaderboardPopup
 
 public:
     void Update(_In_ float fDelta);
+    GSL_SUPPRESS(con.4) // line 249
```

Comment:
putting line numbers in comments is bad practice as code shifts around frequently. in this case, it's not even in the same file.

Reply:
I'll change it